docs/pr_checklist.md

```diff
@@ -55,8 +63,9 @@ https://github.com/qmk/qmk_firmware/pulls?q=is%3Apr+is%3Aclosed+label%3Akeyboard
         - `layout` definitions should include matrix positions, so that `LAYOUT` macros can be generated at build time
             - should use standard definitions if applicable
             - use the Community Layout macro names where they apply (preferred above `LAYOUT`/`LAYOUT_all`)
-            - use of `LAYOUT_all` is only valid when providing additional layout macros
-              - providing only `LAYOUT_all` is invalid - especially when implementing the additional layouts within 3rd party tooling
+            - use of `LAYOUT_all` is only valid when providing additional layout macros (i.e. the keyboard supports multiple layouts)
+                - providing only `LAYOUT_all` is invalid - especially when implementing the additional layouts within 3rd party tooling
+                    - use `LAYOUT` instead
```

Comment:
Not sure if this is the right suggestion, as in the case of implementing everything in via, its just as lazy. If a keyboard only supports a single layout (which is not community), then `LAYOUT` is fine. Otherwise the expectation would be multiple layout macros.